pkg/pipelines/config/config_test.go

```diff
@@ -76,78 +84,78 @@ func TestManifestWalk(t *testing.T) {
+// 			{
+// 				Name:   "cicd",
+// 				: true,
+// 			},
+// 			{
+// 				Name: "development",
 
-				Services: []*Service{
-					{Name: "app-1-service-http"},
-					{Name: "app-1-service-test"},
-					{Name: "app-2-service"},
-				},
-				Apps: []*Application{
-					{
-						Name: "my-app-1",
-						ServiceRefs: []string{
-							"app-1-service-http",
-							"app-1-service-test",
-						},
-					},
-					{
-						Name:        "my-app-2",
-						ServiceRefs: []string{"app-2-service"},
-					},
-				},
-			},
-			{
-				Name: "staging",
-				Services: []*Service{
-					{Name: "app-1-service-user"},
-				},
-				Apps: []*Application{
-					{Name: "my-app-1",
-						ServiceRefs: []string{
-							"app-1-service-user",
-						},
-					},
-				},
-			},
-		},
-	}
+// 				Services: []*Service{
+// 					{Name: "app-1-service-http"},
+// 					{Name: "app-1-service-test"},
+// 					{Name: "app-2-service"},
+// 				},
+// 				Apps: []*Application{
+// 					{
+// 						Name: "my-app-1",
+// 						ServiceRefs: []string{
+// 							"app-1-service-http",
+// 							"app-1-service-test",
+// 						},
+// 					},
+// 					{
+// 						Name:        "my-app-2",
+// 						ServiceRefs: []string{"app-2-service"},
+// 					},
+// 				},
+// 			},
+// 			{
+// 				Name: "staging",
+// 				Services: []*Service{
+// 					{Name: "app-1-service-user"},
+// 				},
+// 				Apps: []*Application{
+// 					{Name: "my-app-1",
+// 						ServiceRefs: []string{
+// 							"app-1-service-user",
+// 						},
+// 					},
+// 				},
+// 			},
+// 		},
+// 	}
 
-	v := &testVisitor{paths: []string{}}
-	err := m.Walk(v)
-	if err != nil {
-		t.Fatal(err)
-	}
+// 	v := &testVisitor{paths: []string{}}
+// 	err := m.Walk(v)
+// 	if err != nil {
+// 		t.Fatal(err)
+// 	}
 
-	want := []string{
-		"development/app-1-service-http",
-		"development/app-1-service-test",
-		"development/app-2-service",
-		"development/my-app-1",
-		"development/my-app-2",
-		"envs/development",
-		"staging/app-1-service-user",
-		"staging/my-app-1",
-		"envs/staging",
-		"cicd/development/app-1-service-http",
-		"cicd/development/app-1-service-test",
-		"cicd/development/app-2-service",
-		"cicd/staging/app-1-service-user",
-		"envs/cicd",
-	}
+// 	want := []string{
+// 		"development/app-1-service-http",
+// 		"development/app-1-service-test",
+// 		"development/app-2-service",
+// 		"development/my-app-1",
+// 		"development/my-app-2",
+// 		"envs/development",
+// 		"staging/app-1-service-user",
+// 		"staging/my-app-1",
+// 		"envs/staging",
+// 		"cicd/development/app-1-service-http",
+// 		"cicd/development/app-1-service-test",
+// 		"cicd/development/app-2-service",
+// 		"cicd/staging/app-1-service-user",
+// 		"envs/cicd",
+// 	}
 
-	if diff := cmp.Diff(want, v.paths); diff != "" {
-		t.Fatalf("tree files: %s", diff)
-	}
-}
+// 	if diff := cmp.Diff(want, v.paths); diff != "" {
+// 		t.Fatalf("tree files: %s", diff)
+// 	}
+// }
 
 func TestEnviromentSorting(t *testing.T) {
```

Comment:
You can now simplify this entirely, we only need one case, `testEnv` can go away.